lib/lightning_web/channels/channel_helpers.ex

```diff
@@ -0,0 +1,18 @@
+defmodule LightningWeb.ChannelHelpers do
+  @moduledoc """
+  Helper functions for channels
+  """
+
+  def reply_with(socket, {:error, error}) do
+    send_error_to_sentry(socket, error)
+    {:reply, {:error, error}, socket}
+  end
+
+  def reply_with(socket, reply) do
+    {:reply, reply, socket}
+  end
+
+  defp send_error_to_sentry(socket, error) do
+    Sentry.capture_message("#{socket.channel}.Error", extra: %{error: error})
```

Comment:
Locally it might work well (always receving an `error` on the string or map types) but as helper function, it can have a render function or inspect the `error` to make sure a Struct or Schema can be rendered.